hydradx/notebooks/Omnipool/LP_returns_analysis3.ipynb

```diff
@@ -304,18 +308,15 @@
   },
```

Comment:
This text still mentions INTR

Reply:
@jepidoptera this still isn't fixed, but I'm going to approve anyway and make a separate ticket to fix it, because I don't wanna keep going back and forth.